Core/src/org/sleuthkit/autopsy/report/ReportHTML.java

```diff
@@ -254,8 +254,8 @@ private String useDataTypeIcon(String dataType) {
                 case TSK_REMOTE_DRIVE:
                     in = getClass().getResourceAsStream("/org/sleuthkit/autopsy/report/images/drive_network.png"); //NON-NLS
                     break;
-                case TSK_CREDIT_CARD_ACCOUNT:
-                    in = getClass().getResourceAsStream("/org/sleuthkit/autopsy/report/images/credit-card.png"); //NON-NLS
+                case TSK_ACCOUNT:
+                    in = getClass().getResourceAsStream("/org/sleuthkit/autopsy/report/images/account_menu.png"); //NON-NLS
```

Comment:
because that was the name of the file in the icon library it came from.  I will change it to something more generic.